common/src/stack/command/stack/commands/remove/pallet/__init__.py

```diff
@@ -102,17 +89,17 @@ def run(self, params, args):
 
 		self.endOutput(padChar='')
 
-	def clean_pallet(self, roll, version, release, OS, arch):
+	def clean_pallet(self, name, version, release, arch, OS):
```

Comment:
Probably cleaner to just pass `pallet` in and reference the variables as pallet.name, pallet.arch, etc. That way I don't need to rely on the order of the Pallet fields.